webapp/client/tools/listUpDownGenes.js

```diff
@@ -103,7 +103,7 @@ Template.listUpDownGenes.events({
 Template.previouslyRunUpDownGenes.onCreated(function() {
   let instance = this;
 
-  instance.subscribe("jobsOfType", "UpDownGenes");
+  instance.subscribe("upDownGenesJobs");
```

Comment:
This is so that we only load relevant fields of the job object. The job's output object has gigantic arrays that significantly slow down the UI.